src/client/app.coffee

```diff
@@ -75,23 +78,59 @@ hexTo3d = ([hexX, hexY]) ->
 uuidToHex = new Map()
 hexToUuid = new Map()
 validHexes = new Set()
+uuidToTile = new Map()
+
+
+# to do: tile manager class
```

Comment:
Yeah, providing an abstraction so we can just deal with the hex coordinates and not the underlying mesh uuids.

Hide away structures like:

    uuidToHex = new Map()
    hexToUuid = new Map()
    validHexes = new Set()
    uuidToTile = new Map()